backend/fregepoc/repositories/views.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import viewsets
+from rest_framework.permissions import DjangoModelPermissions
+
+from fregepoc.repositories.models import Repository, RepositoryFile
+from fregepoc.repositories.serializers import RepositorySerializer, RepositoryFileSerializer
+
+
+class RepositoryViewSet(viewsets.ModelViewSet):
+    queryset = Repository.objects.all()
+    serializer_class = RepositorySerializer
+    permission_classes = [DjangoModelPermissions]
+    filterset_fields = ['analyzed']
+
+
+class RepositoryFileViewSet(viewsets.ModelViewSet):
+    queryset = RepositoryFile.objects.all()
+    serializer_class = RepositoryFileSerializer
+    permission_classes = [DjangoModelPermissions]
+    filterset_fields = ['repository', 'analyzed', 'language']
```

Comment:
We don't want to allow the edition/addition/deletion of the `Repository` & `RepositoryFile` instances via the API, this needs to be switched to `ReadOnlyModelViewSet`.